.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: elixir
+
+elixir: 
+  - '1.10'
+
+otp_release:
+  - '22.3'
+
+services:
+  - postgresql
```

Comment:
I know that we will eventually want a PostgreSQL db for our tests, but I'm curious if your tests currently use a DB at all, i.e., if your test suite would still pass without lines 8-9, then we shouldn't include it here.

Reply:
Ugh. Those lines bring back painful feelings from the hours of debugging it took to get Travis to work.

My tests will pass without those lines, but the build errors out. As far as I understand it, it has something to do with Phoenix generating a DB as soon as a new project is created.